src/models/attestor.models.ts

```diff
@@ -24,7 +24,13 @@ export enum XRPLAttestorChainID {
   'ripple-xrpl-devnet' = 'ripple-xrpl-devnet',
 }
 
-export type AttestorChainID = EVMAttestorChainID | XRPLAttestorChainID;
+export enum StarknetAttestorChainID {
+  'starknet-mainnet' = 'starknet-mainnet',
+  'starknet-testnet' = 'starknet-testnet',
```

Comment:
In the attestor-stack, we renamed this to starknet-sepolia, and starknet-devnet.